csi.proto

```diff
@@ -376,6 +382,13 @@ message CreateVolumeRequest {
   // VOLUME_ACCESSIBILITY_CONSTRAINTS plugin capability, the SP MAY
   // choose where the provisioned volume is accessible from.
   TopologyRequirement accessibility_requirements = 7;
+
+  // Plugins specific mutable parameters passed in as opaque key-value
+  // pairs. This field is OPTIONAL. The Plugin is responsible for 
+  // parsing and validating these parameters. COs will treat these 
+  // as opaque. Plugins SHOULD treat these as if they were applied 
+  // after the creation time parameters.
```

Comment:
"were applied after" suggests a timed sequence of events, which doesn't really happen at create-time - right? or .. can mutable parameters not be provided at create-time?

wondering if changing wording to "override" or "take precedence over" would be more clear?

Reply:
@bswartz What is your thought on this? Whether the mutable parameters is required at creation time or not depends on the provider's volume creation API.

Reply:
I intentionally was referring to a sequence of events, because it's possible to call ModifyVolume immediately after CreateVolume, and I expect the results to be idempotent if the mutable parameters haven't changed.